src/logic/calculate.js

```diff
@@ -0,0 +1,29 @@
+import operate from './operate';
+
+const calculate = (data, buttonName) => {
+  const newData = data;
+
+  switch (buttonName) {
```

Comment:
I guess we need to add some conditions right;
1. `case` '='
2. `case` 'digit'

That's; when `buttonName` = a digit or = sign.